src/components/Screen.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Screen() {
+  return (
+    <div className="screen">
+      <input type="text" placeholder="0" />
+    </div>
```

Comment:
### Suggested Changes

- [ ] The calculator result view's height is too high and does not match the original design. What if you could reduce the height to match the original design?

This is not much of a big change and hence I decided to make a suggestion, I highly recommend you work on it please 👍🏽

**The Original Design**

<img width="344" alt="Screenshot 2023-07-04 at 9 44 11 AM" src="https://github.com/rbd3/math-magicians/assets/50941074/6a5d10e7-5c12-4ffd-8801-1c2e24cd1b58">

**Your Design**

<img width="551" alt="Screenshot 2023-07-04 at 9 44 21 AM" src="https://github.com/rbd3/math-magicians/assets/50941074/df49f698-d3d2-4184-a249-4106203493d7">